units/src/fee_rate.rs

```diff
@@ -328,6 +328,9 @@ mod tests {
     fn fee_rate_from_sat_per_vb_test() {
         let fee_rate = FeeRate::from_sat_per_vb(10).expect("expected feerate in sat/kwu");
         assert_eq!(FeeRate(2500), fee_rate);
+
+        let fee_rate = FeeRate::from_sat_per_kvb(10);
+        assert_eq!(FeeRate(2), fee_rate);
     }
```

Comment:
I know its simple but this breaks the test name, can we put this in a separate test?

```rust
    fn fee_rate_from_sat_per_kvb() {
        let fee_rate = FeeRate::from_sat_per_kvb(10);
        assert_eq!(FeeRate(2), fee_rate);
    }
```

For bonus points put a patch up front that removes the `_test` suffix from the other one.

Reply:
I was being lazy because it's such a trivial test case.  I've added it it's own test.